paper/paper.md

```diff
@@ -120,20 +120,20 @@ Biologically active particles can also be used to model OCDR deployment strategi
 Figure made with ``Makie.jl`` [@makie]. \label{column}](column_example.png)
 
 The implementation of OceanBioME.jl models allows for seamless integration with data assimilation packages, such as ``EnsembleKalmanProcesses.jl`` [@ekp]. 
-This feature facilitates rapid calibration of model parameters, providing a powerful utility for integrating observations and models, with the potential to improve model skill and identify key sources of uncertainty.
+This enables rapid calibration of model parameters and provides a powerful utility for integrating observations and models, with the potential to improve model skill and identify key sources of uncertainty.
 
 A key metric for the validity of biogeochemical systems is the conservation of elements such as carbon and nitrogen in the system.
-We therefore continuously test the implemented models in a variety of simple scenarios (i.e. isolated, with/without air-sea flux, with/without sediment) to ensure that conservation conditions are met, and we will continue to add tests for any new models.
+We therefore continuously test the implemented models in a variety of simple scenarios (i.e. isolated, with/without air-sea flux, with/without sediment) to ensure that conservation conditions are met, and we will continue to add tests for any models.
 Additionally, we check ``OceanBioME.jl`` utilities through standard tests such as comparison to analytical solutions for light attenuation, and conservation of tracers for active particle exudation and sinking.
 We plan to expand the features of ``OceanBioME.jl`` in the future and very much welcome user contributions.
 
-Finally, this software is currently facilitating multiple research projects into ocean CDR which would have been significantly harder with other solutions.
-For example, Chen (In prep.) uses the active particle coupling provided to investigate the effects of location and planting density of kelp in the open ocean on their carbon drawdown effect, as in the example above.
-Additionally, Strong-Wright (In prep.) uses the coupling of both the biogeochemistry and easy interface to couple the physics to study flow interactions with a fully resolved giant kelp forest model including the effects on nutrient transport and distribution.
+Finally, this software is currently facilitating multiple research projects into ocean CDR which would have been significantly harder with other models.
+For example, Chen et al. (in prep.) uses the active particle coupling provided to investigate the effects of location and planting density of kelp in the open ocean on their carbon drawdown effect, as in the example above.
+Strong-Wright et al. (n prep.) uses the coupling of both the biogeochemistry and easy interface to couple the physics to study flow interactions with a fully resolved giant kelp forest model including the effects on nutrient transport and distribution.
```

Comment:
is there anything we could cite here? No worries if not... But even a GitHub repo might be helpful because then the GitHub issue will include the proper citation of these papers when they are out and people would be able to find them in the future.

Reply:
My code for that paper is in a git repo but its not tidy or public yet, perhaps I should just make a place holder one that we can link to and then fill it when its ready?

Reply:
Or do nothing also and leave as is -- it was just a suggestion.

Reply:
We can definitely merge this PR now with "in prep." and if you decide to do something add the ref?

Reply:
good catch, thanks.  Will merge now